.github/workflows/superlinter.yml

```diff
@@ -39,4 +39,3 @@ jobs:
           VALIDATE_RENOVATE: true
           VALIDATE_SHELL_SHFMT: true
           VALIDATE_XML: true
-          VALIDATE_YAML: true
```

Comment:
Let's explicitly `false` this with a comment.

Reply:
This is not supported. Ref: https://github.com/prometheus-community/helm-charts/actions/runs/12852184070/job/35833838668?pr=5209#step:4:210

> Error: -19 09:15:41 [ERROR]   Behavior not supported, please either only include (VALIDATE=true) or exclude (VALIDATE=false) linters, but not both



But I can add a comment before VALIDATE_YAML, if you want.

Reply:
Ugh.